client/src/main/java/io/pravega/client/admin/StreamManager.java

```diff
@@ -9,8 +9,10 @@
  */
 package io.pravega.client.admin;
 
+import com.google.common.annotations.Beta;
 import io.pravega.client.ClientConfig;
 import io.pravega.client.admin.impl.StreamManagerImpl;
+import io.pravega.client.batch.StreamInfo;
```

Comment:
I don't really like this. It is a bit odd that we are returning a data structure under `batch` here. The options I see are:

1- `StreamInfo` should be a first-class data structure and be under `io.pravega.client`
2- We duplicate it in `io.pravega.client.admin` and annotate the one in `io.pravega.client.batch` as deprecated so that we don't have to maintain them both.

Reply:
yes, we should not return a data structure under batch package. Refactored the code to fix it.